shared-functions

```diff
@@ -92,15 +92,30 @@ function bosh_interpolate() {
     arguments="${arguments} -l vars-files/${vf}"
   done
 
-  if [ -n "${release_name}" ]; then
+  if [ -z "${RELEASE_TARBALL}" ]; then
+    if [ -n "${release_name}" ]; then
+      cat << EOF > create-provided-release.yml
+  ---
+  - type: replace
+    path: /releases/name=${release_name}
+    value:
+      name: ${release_name}
+      version: create
+      url: file://${root_dir}/release
+EOF
+      arguments="${arguments} -o create-provided-release.yml"
+    fi
+  else
+    RELEASE_TARBALL=$(readlink -f -- ${RELEASE_TARBALL})
```

Comment:
we were using `realpath` but it seems it's not there by default and is installed as part of the core-utils on ubuntu. I think the default docker image may not have it, so `readlink` works. 
it allows us to complete paths that are like `../diego-release/*.tgz` to `/tmp/workspace/diego/diego-release.2.2.2.tgz` for example.